mdspyutils/mdspyutils.py

```diff
@@ -2,13 +2,18 @@
 
 
 def parse_lab(file_name: str) -> list:
-    """[summary]
+    """This function parse the content of MDS lab files
```

Comment:
"parses"

Reply:
For now, we are only giving the option to parse markdown content right? If so, I would add that to the title.

Reply:
3a5efa9 resolved via Kamal.